cmd/kk/pkg/kubernetes/tasks.go

```diff
@@ -154,11 +154,11 @@ func SyncKubeBinaries(runtime connector.Runtime, binariesMap map[string]*files.K
 	return nil
 }
 
-type SyncKubelet struct {
+type ChownKubelet struct {
 	common.KubeAction
 }
 
-func (s *SyncKubelet) Execute(runtime connector.Runtime) error {
+func (c *ChownKubelet) Execute(runtime connector.Runtime) error {
 	if _, err := runtime.GetRunner().SudoCmd("chmod +x /usr/local/bin/kubelet", false); err != nil {
 		return errors.Wrap(errors.WithStack(err), "sync kubelet service failed")
```

Comment:
Maybe this error log can also be modified.

Reply:
> Maybe this error log can also be modified.

Sorry for that, I'll re-push the pr to complete the log of error.